en/guide/mavlink_router.md

```diff
@@ -0,0 +1,276 @@
+# MAVLink Router
+
+The [mavlink-router](https://github.com/mavlink-router/mavlink-router) can be used to route and filter MAVLink packets between UART, UDP and TCP endpoints, and to log MAVLink traffic.
+
+This topic provides an overview of what the router can do, and how it is used.
+See the [official docs](https://github.com/mavlink-router/mavlink-router) for more information, including instructions for building and installation.
+
+
+## Using mavlink-router
+
+MAVLink router can be started in a terminal using the `mavlink-routerd` command.
+The endpoints and other options can be configured using [command line arguments](#command-line-syntax) or a [configuration file](#configuration-file).
+
+A simple usage is to specify a single endpoint using the `e` flag, and then the routed interface: 
+
+```bash
+mavlink-routerd -e 10.73.41.30:14550 127.0.0.1:14550
+```
+
+A number of examples are provided below
+
+
+### Routing from a UART to other endpoints
+
+To route mavlink packets from `ttyS1` to two UDP endpoints (from the official docs!):
+
+```bash
+mavlink-routerd -e 192.168.7.1:14550 -e 127.0.0.1:14550 /dev/ttyS1:1500000
```

Comment:
Traffic is also routed between 192.168.7.1:14550 and 127.0.0.1:14550 if a packet that is send to a system/component available in that endpoint (or if it's a broadcast).

I'm preferring to document it like:
mavlink-routerd -e 192.168.7.1:14550 -e 127.0.0.1:14550 -e /dev/ttyS1:1500000

so it's clear they are all "Endpoints". I realize this is not what the README.md is currently saying. Maybe one thing that could help is to show how the command line options translate to config, which is maybe clearer to understand. In this example it would be:

```
[UartEndpoint uart0]
Device = /dev/ttyS1
Baud = 1500000

[UdpEndpoint udp0]
Address = 192.168.7.1
Port = 14550

[UdpEndpoint udp1]
Address = 127.0.0.1
Port = 14550
```